### PR TITLE
Fix build errors with musl C library.

### PR DESCRIPTION
**Pull request by bkuhls**
Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

Next try ;) https://github.com/OpenLightingProject/ola/pull/1025#issuecomment-177539085

### CODE REVIEW

File: plugins/spi/SPIWriter.cpp [hunk 1]
```diff
@@ -18,6 +18,7 @@
  * Copyright (C) 2013 Simon Newton
  */
 
+#include <linux/ioctl.h>
```
Comment (peternewman) [inline plugins/spi/SPIWriter.cpp]: Can you put this above linux/spi/spidev.h please, unless it has to be right at the top for some reason?

**Comment by bkuhls** [comment]
Re-ordered include in plugins/spi/SPIWriter.cpp

**Comment by peternewman** [comment]
LGTM. You don't need to bother flattening next time.

**Comment by peternewman** [comment]
Retrospective if you want @nomis52 .

**Comment by nomis52** [comment]
LGTM

**Comment by peternewman** [comment]
Hi @bkuhls it seems your change broke our Mac build, among others, I've created a PR to fix it, https://github.com/OpenLightingProject/ola/pull/1030 . I've no reason to doubt it won't continue to work on your platform, but you may wish to give it a quick test to make sure.

**Pull request merged**